device/esp_tinyusb/Kconfig

```diff
@@ -41,7 +41,7 @@ menu "TinyUSB Stack"
 
         choice TINYUSB_TASK_AFFINITY
             prompt "TinyUSB task affinity"
-            default TINYUSB_TASK_AFFINITY_NO_AFFINITY
+            default TINYUSB_TASK_AFFINITY_CPU1
             depends on !TINYUSB_NO_DEFAULT_TASK
             help
```

Comment:
This is what I can see now for ESP32-S2

![image](https://github.com/user-attachments/assets/82617764-0322-40f0-8ad2-19523a7f3806)

Selecting task affininty on single-core system might confuse the users (Why should I choose CPU affinity if there is only 1 CPU?). 

Could you please fix this so the 'TinyUSB task affinity' menu `depends on !FREERTOS_UNICORE` ?

Reply:
I hate the invert logic (especially, `bool "Do not create a TinyUSB task"` part, which led in the code to `#if !CONFIG_TINYUSB_INIT_IN_DEFAULT_TASK`). I want to refactor everything. But later. 

I created a ticket for that and right now just made a dependency of default parameter from `FREERTOS_UNICORE` not to change the config too much.  Because for `esp32s2` the AFFINITY_MASK is absolutely pointless.
:angry: